db/migrate/20240924105524_create_branches.rb

```diff
@@ -0,0 +1,14 @@
+class CreateBranches < ActiveRecord::Migration[7.2]
+  def change
+    create_table :branches do |t|
+      t.integer :store_id
+      t.integer :user_id
+      t.string :name
+      t.string :address
+      t.string :phone
+      t.string :status
```

Comment:
```suggestion
      t.integer :status
```
tương tự mấy chỗ khác đi, e chưa biết enum hả?

Reply:
Dạ, em có biết ạ, em sẽ sửa ạ.